tests/test_001_model.py

```diff
@@ -18,20 +20,17 @@ def test_model_directories(tmpdir):
 def get_ds_time_steady(tmpdir, modelname="test"):
     model_ws = os.path.join(tmpdir, "test_model")
     ds = nlmod.base.set_ds_attrs(xr.Dataset(), modelname, model_ws)
-    ds = nlmod.time.set_ds_time(ds, start_time="2015-1-1", steady_state=True)
+    ds = nlmod.time.set_ds_time(ds, ["2015-1-2"], start="2015-1-1", steady=True)
     return ds
 
 
 def get_ds_time_transient(tmpdir, modelname="test"):
     model_ws = os.path.join(tmpdir, "test_model")
     ds = nlmod.base.set_ds_attrs(xr.Dataset(), modelname, model_ws)
-    ds = nlmod.time.set_ds_time(
-        ds,
-        start_time="2015-1-1",
-        steady_state=False,
-        steady_start=True,
-        transient_timesteps=10,
-    )
+    nper = 11
+    time = pd.date_range(start="2015-1-1", periods=nper, freq="D")
+    steady = np.zeros(nper)
+    ds = nlmod.time.set_ds_time(ds, time, start="2015-1-1", steady=steady)
```

Comment:
I think the `start` should not be the same as the first item in `time` since `time` is the time at the end of each period.